include/zephyr/net/net_linkaddr.h

```diff
@@ -146,6 +131,78 @@ static inline int net_linkaddr_set(struct net_linkaddr_storage *lladdr_store,
 	return 0;
 }
 
+/**
+ * @brief Copy link address from one variable to another.
+ *
+ * @param dst The link address structure destination.
+ * @param src The link address structure to source.
+ * @return 0 if ok, <0 if error
+ */
+static inline int net_linkaddr_copy(struct net_linkaddr *dst,
+				    const struct net_linkaddr *src)
+{
+	if (dst == NULL || src == NULL) {
+		return -EINVAL;
+	}
+
+	if (src->len > NET_LINK_ADDR_MAX_LENGTH) {
+		return -EMSGSIZE;
+	}
+
+	dst->type = src->type;
+	dst->len = src->len;
+	memcpy(dst->addr, src->addr, src->len);
+
+	return 0;
+}
+
+/**
+ * @brief Clear link address.
+ *
+ * @param lladdr The link address structure.
+ * @return 0 if ok, <0 if error
+ */
+static inline int net_linkaddr_clear(struct net_linkaddr *lladdr)
```

Comment:
This function could just call the create function with the src members as parameters